scripts/cluster-scanner/internal/search_old_clusters.go

```diff
@@ -0,0 +1,32 @@
+package internal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spectrocloud/palette-sdk-go/api/models"
+)
+
+// SearchOldClusters checks for clusters that have been running for more than 24 hours.
+// It takes a list of cluster summaries and returns a slice of messages with the
+// information of the clusters that were found.
+func SearchOldClusters(clusters []*models.V1SpectroClusterSummary) ([]string, error) {
+	// Slice of strings to keep track of any found clusters older than 24 hours
+	var messageArray []string
+
+	// Iterate through the clusters to find those running for more than 24 hours
+	for _, cluster := range clusters {
+		timeValue := time.Time(cluster.Metadata.CreationTimestamp)
+		clusterAge := time.Now().Sub(timeValue)
```

Comment:
This is 100% valid - but if you want to be hip ✨, then you can use the new function released recently [`time.Since(t)` ](https://pkg.go.dev/time#Since)

Reply:
Cool! Function replaced :white_check_mark: